src/main/java/org/prebid/cache/model/PayloadTransfer.java

```diff
@@ -26,4 +27,30 @@ public String getValue() {
             return Json.toJson(value);
         }
     }
+
+    @Override
```

Comment:
I think this is redundant since lombok's `@Value` takes care of generating these methods. Is there a good reason to replace them with own implementation?

Reply:
there is an issue with tests because of logic in Value gatter